contrib/git-grab/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "git-grab"
+pkgver = "2.1.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo"]
+makedepends = [
+    "rust-std",
+]
```

Comment:
```suggestion
makedepends = ["rust-std"]
```

when it's 1 you can leave it single line

Reply:
Fixed now

Reply:
black doesn't have an opinion about it (if you have trailing , it always leaves it)